datagrid_gtk3/utils/transformations.py

```diff
@@ -105,6 +106,12 @@ def string_transform(value, max_length=None, oneline=True,
                 raise
             value = decode_fallback(value)
 
+    # Remove non-printable caracteres from the string. Only keep those
+    # considered as whitespace/newline. We cannot use string.printable
+    # here as it would remove unicode caracteres too.
+    value = u''.join(c for c in value
+                     if ord(c) >= 32 or c in string.whitespace)
```

Comment:
@tomanderson2 Yes, it sounds reasonable. Sorry, I forgot for a moment which repository this is... :/